wix/Build.OpenJDK_generic.cmd

```diff
@@ -166,7 +162,7 @@ FOR %%A IN (%ARCH%) DO (
 
     ECHO Source dir used : !REPRO_DIR!
 
-    SET OUTPUT_BASE_FILENAME=!PRODUCT_SKU!!PRODUCT_MAJOR_VERSION!-!PRODUCT_CATEGORY!_!FOLDER_PLATFORM!_windows_!PACKAGE_TYPE!-!PRODUCT_VERSION!
+    SET OUTPUT_BASE_FILENAME=!PRODUCT_SKU!!PRODUCT_MAJOR_VERSION!-!PRODUCT_CATEGORY!_!FOLDER_PLATFORM!_windows_!PACKAGE_TYPE!-!PRODUCT_FULL_VERSION!
```

Comment:
as said, the OUTPUT_BASE_FILENAME is not a concern as it the msi is always renamed with the zip name
but ok :)

Reply:
I remember, I just wanted to cover all our bases. :)